### DISCUSSION TITLE
Filter list of Databases

### DISCUSSION THREAD

**Discussion by gwired2015** [dbeaver/dbeaver#12777] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
My database has over 5k objects and hits the limit and it is hard to search for the schema/table I want via scrolling.

**Describe the solution you'd like**
A Tables filter similar to Columns filter.  However,  I would like to be able to filter on properties of database in the server like is a User Table or not.

**Describe alternatives you've considered**
None known or considered for a solution.

**Additional context**
N/A

**Comment by uslss** [comment, marked as answer]
databases/schemas can be filtered in Connection properties
![image](https://user-images.githubusercontent.com/31996417/121176548-02d99c80-c865-11eb-8324-2663c656df29.png)
